applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -175,7 +188,7 @@ static bool is_page_clean(const struct flash_area *fa, off_t off, size_t len)
 		}
 
 		for (size_t j = 0; j < ARRAY_SIZE(buf); j++) {
-			if (buf[j] != FLASH_CLEAN_VAL) {
+			if (buf[j] != erased_val) {
```

Comment:
Will it work? `erased_val` will be equal to `0xFF` as far as I understand as the used function returns the `uint8_t`.

Reply:
Example from mcumgr how they do it:
```
#define ERASED_VAL_32(x) (((x) << 24) | ((x) << 16) | ((x) << 8) | (x))

uint8_t erased_val;
uint32_t erased_val_32;

erased_val = flash_area_erased_val(fa);
erased_val_32 = ERASED_VAL_32(erased_val);
```